src/scripts/i18n/list_source_files.py

```
@@ -0,0 +1,113 @@
+
+def gather_source_files(types: List[str]) -> Dict[str, str]:
+    """Return mapping of file_path → content for the requested types."""
+    # Build list of suffixes we care about.
+    suffixes: List[str] = []
+    for t in types:
+        suffixes.extend(SUFFIX_MAP[t])
+
+    # Mobile app has a single src/app structure
+    root = "src/app"
+
+    files: Dict[str, str] = {}
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)